store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+                if value.lower() in ('true', 'false'):
+                    fields[key.strip()] = value.lower() == 'true'
+                elif value.isdigit():
+                    fields[key.strip()] = int(value)
+                elif value.replace('.', '').isdigit():
+                    fields[key.strip()] = float(value)
+                else:
+                    fields[key.strip()] = value
+        return fields if fields else {"message": "Hello from Firestore!"}
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)